bitnami/prometheus-operator/templates/prometheus/prometheus.yaml

```diff
@@ -114,7 +114,7 @@ spec:
         resources:
           requests:
             storage: {{ .Values.prometheus.persistence.size | quote }}
-        {{- include "prometheus-operator.prometheus.storageClass" . }}
+        {{ include "prometheus-operator.prometheus.storageClass" . }}
```

Comment:
Please could you use `| indent x`?

Reply:
@dani8art, I have updated the PR to use `nindent` (reverting the leading newline trimming).